app/reducers/snackbar.js

```diff
@@ -419,15 +424,21 @@ export default function snackbar(state = {}, action) {
   case TRZ_UPDATEFIRMWARE_FAILED:
   case TRZ_NOCONNECTEDDEVICE:
   case TRZ_GETWALLETCREATIONMASTERPUBKEY_FAILED:
+    type = "Error";
     if (action.error && String(action.error).indexOf("wallet.Unlock: invalid passphrase:: secretkey.DeriveKey") > -1) {
       // intercepting all wrong passphrase errors, independently of which error
       // state was triggered. Not terribly pretty.
       message = messages[WRONG_PASSPHRASE_MSG];
+    } else if (String(action.error).indexOf("[object Object]") > -1) {
```

Comment:
Looking again, it's probably better to test whether this is actually an object (`typeof action.error == "object"`) vs coercing it into a string.

Reply:
I first made like this, the problem with this approach, is that some errors are thrown as objects, so they and on this condition. For example: 

```
Error: 2 UNKNOWN: wallet.PublishUnminedTransactions:: dcrd.jsonrpc.sendrawtransaction: -40: Rejected transaction 4c47d7808aa267f430d2c82a2ff1b78ff1008e2b1cabab0c40bb08c5edec521a: already have transaction 4c47d7808aa267f430d2c82a2ff1b78ff1008e2b1cabab0c40bb08c5edec521a
    at Object.exports.createStatusError (/home/vctt/projects/decred/decrediton/app/node_modules/grpc/src/common.js:91)
    at Object.onReceiveStatus (/home/vctt/projects/decred/decrediton/app/node_modules/grpc/src/client_interceptors.js:1204)
    at InterceptingListener._callNext (/home/vctt/projects/decred/decrediton/app/node_modules/grpc/src/client_interceptors.js:568)
    at InterceptingListener.onReceiveStatus (/home/vctt/projects/decred/decrediton/app/node_modules/grpc/src/client_interceptors.js:618)
    at callback (/home/vctt/projects/decred/decrediton/app/node_modules/grpc/src/client_interceptors.js:845)
```

And when making typeof this error we get an object.

![image](https://user-images.githubusercontent.com/15069783/58175181-e9615a00-7c75-11e9-8903-5984240e3a4e.png)

Reply:
Got it